src/ontology/edits/oeo-physical.omn

```diff
@@ -2871,6 +2872,54 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130",
         <http://openenergy-platform.org/ontology/oeo/OEO_00000316>
     
     
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00230000>
+
+    Annotations: 
+        <http://openenergy-platform.org/ontology/oeo/definition> "Storage capacity is the maximum energy an energy storage object can store."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/320"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "pull request: https://github.com/OpenEnergyPlatform/ontology/pull/391",
+        rdfs:label "storage capacity"@en
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000150>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00230001>
+
+    Annotations: 
+        <http://openenergy-platform.org/ontology/oeo/definition> "Power rating is the maximum power an energy converting device can convert."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/320"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "pull request: https://github.com/OpenEnergyPlatform/ontology/pull/391",
+        rdfs:label "power rating"@en
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000333>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00230002>
+
+    Annotations: 
+        <http://openenergy-platform.org/ontology/oeo/definition> "Declared net capacity is the maximum power a power plant can deliver to the electrical grid. It equals the sum of the rated powers of all plant generators minus all power used internally within the plant."@en,
```

Comment:
According to this definition the term _declared net capacity_ is only applicable to _power plants_ but not to _power generating units_. Is this intended?

Reply:
> Is this intended?

Nope.  Added them.